msteams-platform/messaging-extensions/how-to/link-unfurling.md

```diff
@@ -21,6 +21,54 @@ The Azure DevOps message extension uses link unfurling to look for URLs pasted i
 
 :::image type="content" source="~/assets/images/compose-extensions/messagingextensions_linkunfurling.png" alt-text="Example of link unfurling":::
 
+## App less adaptive card preview with schema.org templates
+
+Schema.org is a collaborative, community based activity with a mission to create, maintain, and promote schemas for the structured data on the internet, web pages, email and beyond.
+
+### How does it work
+
+When metadata of the webpage is tagged to a type of product it unfurls to an adaptive card. App type plays an crucial role, each type will have a designated template. The adaptive card is upgraded with lot more action buttons
```

Comment:
> The adaptive card is upgraded with lot more action buttons

Where do the action buttons come from?
(Also, "a lot more" feels a bit colloquial, worth checking the style guide)

Reply:
removed the sentence.